src/referencePackages/src/nuget.common/6.13.2/lib/netstandard2.0/NuGet.Common.cs

```diff
@@ -498,12 +498,12 @@ public static partial class NuGetEventSource
 
         public static partial class Keywords
         {
-            public const System.Diagnostics.Tracing.EventKeywords Common = 1L;
-            public const System.Diagnostics.Tracing.EventKeywords Configuration = 2L;
-            public const System.Diagnostics.Tracing.EventKeywords Logging = 4L;
-            public const System.Diagnostics.Tracing.EventKeywords Performance = 8L;
-            public const System.Diagnostics.Tracing.EventKeywords Restore = 32L;
-            public const System.Diagnostics.Tracing.EventKeywords SdkResolver = 16L;
+            public const System.Diagnostics.Tracing.EventKeywords Common = (System.Diagnostics.Tracing.EventKeywords)1L;
+            public const System.Diagnostics.Tracing.EventKeywords Configuration = (System.Diagnostics.Tracing.EventKeywords)2L;
+            public const System.Diagnostics.Tracing.EventKeywords Logging = (System.Diagnostics.Tracing.EventKeywords)4L;
+            public const System.Diagnostics.Tracing.EventKeywords Performance = (System.Diagnostics.Tracing.EventKeywords)8L;
+            public const System.Diagnostics.Tracing.EventKeywords Restore = (System.Diagnostics.Tracing.EventKeywords)32L;
+            public const System.Diagnostics.Tracing.EventKeywords SdkResolver = (System.Diagnostics.Tracing.EventKeywords)16L;
```

Comment:
@MichaelSimons - As far as source modifications, what are we supposed to do in this case? These customizations were made to the generated file. Is there any other option here?

Reply:
This source modification is the only option.  It is the pattern used for the other nuget package versions in the repo.  This particular issue is tracked with https://github.com/dotnet/source-build/issues/4328.